horovod/runner/elastic/discovery.py

```diff
@@ -18,10 +18,17 @@
 import threading
 
 from collections import defaultdict
+from enum import Enum
 
 from horovod.runner.common.util import safe_shell_exec
 
 
+class HostUpdateResult(Enum):
```

Comment:
If we put this in `runner.elastic.worker`, we can use this as part of the message sent from driver to workers.